src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size;
+  border: 1px solid #000;
+
+  @each $day, $date in $days {
+    &--start-day--#{$day}:first-child {
+      margin-left: ($cell-size * $date) + ($cell-gap * $date);
+    }
+  }
```

Comment:
you havn't written modifier for `last-date` to make it work
`@for $lastDate from 28 through 31` somthing like this for `month-length` modifier